### PR TITLE
PGD copy edits by Dee Dee

### PR DESCRIPTION
**Pull request by drothery-edb**
## What Changed?

Restructured Deployment options section
Other cleanup

### CODE REVIEW

File: product_docs/docs/pgd/4/deployments/tpaexec/overview.mdx [hunk 1]
```diff
@@ -1,35 +1,44 @@
...
-title: "TPAexec"
+title: "TPAexec overview"
 ---
 
 TAKEN FROM: https://documentation.enterprisedb.com/tpa/release/22.13-1/
 
 TPAexec is an orchestration tool that uses Ansible to build Postgres clusters as specified by TPA (Trusted Postgres Architecture), a set of reference architectures that document how to set up and operate Postgres in various scenarios. TPA represents the best practices followed by EDB (and formerly, 2ndQuadrant), and its recommendations are as applicable to quick testbed setups as to production environments.
 
-What can TPAexec do?
+## What can TPAexec do?
```
Comment (PJMODOS) [inline product_docs/docs/pgd/4/deployments/tpaexec/overview.mdx:9]: This part shows how to setup physical replication cluster, likely not what we want in PGD docs

Comment (PJMODOS) [reply product_docs/docs/pgd/4/deployments/tpaexec/overview.mdx:9]: I might even just scratch this whole "What can TPAexec do?" section, as the  "Capabilities and supported software" describes it better and then we have quick setup which shows how to use that for BDR/PGD

Comment (drothery-edb) [reply product_docs/docs/pgd/4/deployments/tpaexec/overview.mdx]: I removed the section

**Comment by drothery-edb** [comment]
I'm going to merge this into our main release branch.

**Pull request merged**